medusa/service/grpc/client.py

```diff
@@ -100,3 +100,13 @@ def backup_exists(self, name):
         except grpc.RpcError as e:
             logging.error("Failed to determine if backup exists for backup name: {} due to error: {}".format(name, e))
             return False
+
+    def purge_backups(self):
+        try:
+            stub = medusa_pb2_grpc.MedusaStub(self.channel)
+            request = medusa_pb2.PurgeBackupsRequest()
+            resp = stub.PurgeBackups(request)
+            return resp
+        except grpc.RpcError as e:
+            logging.error("Failed to purge backupsdue to error: {}".format(e))
```

Comment:
minor space to include after 'backups'

Reply:
yup, good catch 👍

Reply:
It's actually fixed in the follow up PR that will be merged into this branch before we merge it to `master`: https://github.com/thelastpickle/cassandra-medusa/pull/449/files#diff-1d56ea287055089d1407ae22c124dfb7e4ad2b53d76c22e04a9c346435ddfc55R112